server/hooks/after_copyCartToOrder.js

```diff
@@ -1,72 +1,70 @@
-// function shipmentChecker(date) {
-//   if (moment(date).isoWeekday() === 7) {
-//     return moment(date).subtract(2, 'days')._d;
-//   } else if (moment(date).isoWeekday() === 6) {
-//     return moment(date).subtract(1, 'days')._d;
-//   }
-//   return date;
-// }
+ReactionCore.MethodHooks.after('cart/copyCartToOrder', function (options) {
+  const orderId = options.result || arguments[0];
+  const order = ReactionCore.Collections.Orders.findOne(orderId);
+  const afPackage = ReactionCore.Collections.Packages.findOne({
+    name: 'reaction-advanced-fulfillment',
+    shopId: ReactionCore.getShopId()
+  });
+  let af = {};
+  advancedFulfillment = af.advancedFulfillment = {};
+  advancedFulfillment.workflow = {
+    status: 'orderCreated',
+    workflow: []
+  };
+  advancedFulfillment.items = AdvancedFulfillment.itemsToAFItems(order.items);
+  let orderHasNoRentals = _.every(order.items, function (item) {
+    return item.variants.functionalType === 'variant';
+  });
+  const shippingAddress = AdvancedFulfillment.addressFormatForFedExApi(order.shipping[0].address);
+  // check if local delivery
+  advancedFulfillment.localDelivery = AdvancedFulfillment.determineLocalDelivery(order.shipping[0].address.postal);
+  if (orderHasNoRentals) {
+    let today = new Date();
+    advancedFulfillment.shipmentDate = AdvancedFulfillment.date.nextBusinessDay(today);
+  } else {
+    // if local set tranist time to 0 else call FedEx Api for tranist time
+    advancedFulfillment.transitTime = advancedFulfillment.localDelivery ? 0 : AdvancedFulfillment.determineShippingCarrier(afPackage.settings.selectedShipping, shippingAddress);
+    if (advancedFulfillment.transitTime === false) {
+      advancedFulfillment.transitTime = afPackage.settings.buffer.shipping || 4; // If no fedEx setting to general date
+    }
+    // set all the transit times - as we should have order.startTime and order.endtime
+    // TODO REMOVE!!!!!!!!!mocking random start date
```

Comment:
We can't until we actually pair Rental Orders in, that are coming with start and end time.